docs/content/en/docs/user-guide/configuration-reference.md

```diff
@@ -286,6 +286,7 @@ spec:
 | expected | [AnalysisExpected](/docs/user-guide/configuration-reference/#analysisexpected) | The expected query result. | Yes |
 | interval | duration | Run a query at specified intervals. | Yes |
 | failureLimit | int | Acceptable number of failures. e.g. If 1 is set, the `ANALYSIS` stage will end with failure after two queries results failed. Defaults to 1. | No |
+| nodataAsSuccess | bool | If true, it considers as a success when no data returned from the analysis provider. Defaults to false. | No |
```

Comment:
nits, should it be `noDataAsSuccess` 🤔

Reply:
You're quite right. It should be so. But I just came up with another one named `ignoreNoData`. Let me change to it.

Reply:
That's nice 💯